omero/sysadmins/unix/install-web.txt

```diff
@@ -241,30 +246,27 @@ Customizing your OMERO.web installation
 ---------------------------------------
 
 By default OMERO.web expects to be run from the root URL of the web server.
-This can be changed by setting :property:`omero.web.force_script_name` and
+This can be changed by setting :property:`omero.web.prefix` and
 :property:`omero.web.static_url`. For example, to make OMERO.web appear at
 `http://example.org/omero/`:
 
 ::
 
-    $ bin/omero config set omero.web.force_script_name '/omero'
+    $ bin/omero config set omero.web.prefix '/omero'
     $ bin/omero config set omero.web.static_url '/omero/static/'
 
 and regenerate your web-server configuration
 (see :ref:`fastcgi_configuration`).
 
-OMERO.web is configured to use FastCGI TCP by default. If you are using a
-non-standard web server configuration you may wish to change this:
-
-::
-
-    $ bin/omero config set omero.web.application_server "fastcgi" / "fastcgi-tcp"
+The front-end webserver (Nginx, Apache) can be setup to run on a different
+host from OMERO.web. You will need to set
+:property:`omero.web.application_server.host` to ensure OMERO.web is
+accessible on an external IP.
 
-The default `fastcgi-tcp` option uses a TCP connection for communication
-between OMERO.web and your web server; `fastcgi` uses a file based socket and
-will require you to manage the file permissions on it. You will have to
-regenerate your web server configuration after changing this property
-(see :ref:`fastcgi_configuration`).
+OMERO.web is configured to use FastCGI TCP by default. If you are using a
+non-standard web server configuration and wish to use a file based socket
+you can set :property:`omero.web.application_server` to `fastcgi`, but you
+will have to create your own web server configuration file.
```

Comment:
I'd mention here "and will require you to manage the file permissions on it" and maybe kindly say that is hard?